pages/sidecar.rst

```diff
@@ -70,7 +70,7 @@ CentOS
 Install the Graylog Sidecar repository configuration and Graylog Sidecar itself with the following commands::
 
     $ sudo rpm -Uvh https://packages.graylog2.org/repo/packages/graylog-sidecar-repository-1-2.noarch.rpm
-    $ sudo yum update && sudo yum install graylog-sidecar
```

Comment:
Can you remove the `yum update` from the `pages/installation/os/centos.rst` as well? Thanks!

Reply:
Sure thing!